whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +137,101 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script src="http://rawgit.com/Logicify/jquery-locationpicker-plugin/master/dist/locationpicker.jquery.js"></script>
```

Comment:
Put this in ``base.html``. There are some other changes that should be made there but I'll save those for after this gets merged.